test/textlint-rule-date-weekday-mismatch-test.js

```diff
@@ -2,7 +2,16 @@ import TextLintTester from "textlint-tester";
 
 const tester = new TextLintTester();
 // rule
-const rule = require("../src/textlint-rule-date-weekday-mismatch");
+const dateWeekdayMismatchRule = require("../src/textlint-rule-date-weekday-mismatch");
+const rule = {
+    rules: [{
+        ruleId: "date-weekday-mismatch",
+        rule: dateWeekdayMismatchRule,
+        options: {
+            lang: "ja"
+        }
+    }]
+}
```

Comment:
Was there a problem with passing the rule object as is?
Most test cases test if no options = automatic determination works well..

Reply:
`const rule = ...` is for multiple rules or plugin combination.

Reply:
After updating the dependencies, the tests failed:
```
+ '2016年12月29日(月)'
- '2016年12月29日(木)'
               ^
      + expected - actual

      -2016年12月29日(月)
      +2016年12月29日(木)
```

I assumed this was due to needing the language set?

Reply:
English also failing:
```
+ '2021-07-02 (Thursday)'
- '2021-07-02 (Friday)'
               ^
      + expected - actual
```